bokeh/models/tools.py

```diff
@@ -754,6 +754,13 @@ class ZoomOutTool(ActionTool):
     Percentage to zoom for each click of the zoom-in tool.
     """)
 
+    maintain_focus = Bool(default=True, help="""
+    Whether or not zooming tool maintains a centered focus position. Setting to
```

Comment:
This is misleading. Zoom out will always be centered, regardless of this setting. What can change is the data-space aspect ratio.

Reply:
I am not 100% sure if we have a missunderstanding of what maintain_focus does, or if you simply use different words than me to describe it. To be sure we are on the same page I drew a sketch:

![maintain_focus](https://user-images.githubusercontent.com/10073831/113553459-1b74ce80-95f8-11eb-90c2-888de6f640a2.png)

In my view the focus point (in orange) moves with maintain_focus False, if we get into conflict with the borders, which is why I describe it as is. I will give it a try to describe it more along the lines that you are suggesting in any case.

Reply:
Well, I had not considered so much the case of hitting one side bound before another. I still don't love the name, since focus not apparently means "exactly where the mouse points" for one tool and "implicitly the center of the viewport before the first zoom operation" for another tool, but that just points to bad original decision, I guess. The name should have always referred to the effect of bounds, which could have been consistently applied/explained for all tools. But we are where we are, so I suppose this name will have to do.